pgzero/screen.py

```diff
@@ -72,6 +72,10 @@ def __init__(self, surface):
         self.surface = surface
         self.width, self.height = surface.get_size()
 
+    def bounds(self):
+        """Returns a Rect representing the bounds of the screen"""
```

Comment:
Nitpicking, but this docstring is not PEP-257 compliant - it should be in the imperative mood ("Return" not "Returns") and it should end with a .